### PR TITLE
ENH: Provide a specific error if attr is missing warning about package name.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by danielballan** [comment]
Closes #154

**Comment by ordirules** [comment]
`pip install attr` will still have `import attr` be successful. I don't know if that resolves it
Maybe do something like:
```python
try:
    attr.s
except AttributeError("Error wrong attr library imported. Please uninstall attr and install attrs")
```

I think could be wrong... I think this naming conflict should maybe be resolved higher up.

**Comment by danielballan** [comment]
Good point. Fixed. I think this is the best we can do without asking attr or attrs to break their naming, which seems unlikely.

**Comment by ordirules** [comment]
great, looks good to me.
yeah, it's a shame that nomenclature conflict somehow went through...

thanks!

File: databroker/__init__.py [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 
 try:
+    import attr.s
```
Comment (tacaswell) [inline databroker/__init__.py]: `s` is not an importable thing, needs to be `import attr; attr.s`

**Comment by tacaswell** [comment]
I am not sold on this.  We now have `attrs` in setup.py so the correct one should be installed with data broker.

If the user then goes and `pip install attr` later, that is on the user and I am not sure we should be responsible for saving them.

**Comment by stuartcampbell** [comment]
I think it's not a necessity, but it does seem a nicety for those folks who are not intimate with the software and the various `attr[s]` packages.  For such a small change, it could help someone figure out this potential issue in the future if it arises again.

**Comment by danielballan** [comment]
Point take that fixing the setup.py should be enough that we never see this error again. But I do find it exceptionally confusing. I think it's worth this special check.

It seems that's the consensus above.

**Pull request merged**